pkg/cloudprovider/providers/aws/aws_loadbalancer.go

```diff
@@ -191,6 +192,34 @@ func (s *AWSCloud) ensureLoadBalancer(namespacedName types.NamespacedName, loadB
 		}
 	}
 
+	// Whether the ELB was new or existing, sync attributes regardless. This accounts for things
+	// that cannot be specified at the time of creation and can only be modified after the fact,
+	// e.g. idle connection timeout.
+	{
+		describeAttributesRequest := &elb.DescribeLoadBalancerAttributesInput{}
+		describeAttributesRequest.LoadBalancerName = aws.String(loadBalancerName)
+		describeAttributesOutput, err := s.elb.DescribeLoadBalancerAttributes(describeAttributesRequest)
+		if err != nil {
+			glog.Warning("Unable to retrieve load balancer attributes during attribute sync")
+			return nil, err
+		}
+
+		foundAttributes := &describeAttributesOutput.LoadBalancerAttributes
+
+		// Update attributes if they're dirty
+		if !reflect.DeepEqual(loadBalancerAttributes, foundAttributes) {
+			modifyAttributesRequest := &elb.ModifyLoadBalancerAttributesInput{}
+			modifyAttributesRequest.LoadBalancerName = aws.String(loadBalancerName)
+			modifyAttributesRequest.LoadBalancerAttributes = loadBalancerAttributes
+			_, err = s.elb.ModifyLoadBalancerAttributes(modifyAttributesRequest)
+			if err != nil {
+				glog.Warning("Unable to update load balancer attributes during attribute sync")
+				return nil, err
```

Comment:
This one really should be `return nil,fmt.Errorf("Unable to update load balancer attributes during attribute sync: %v", err)`, and then you can drop the glog.Warning line.  I try always to wrap errors from "external code" i.e. AWS because sometimes they are really cryptic when printed.